web/src/features/cHaines/pages/CHainesPage.tsx

```diff
@@ -0,0 +1,736 @@
+import React, { useRef, useEffect, useState } from 'react'
+import ReactDOMServer from 'react-dom/server'
+import { selectCHainesModelRuns } from 'app/rootReducer'
+import { useDispatch, useSelector } from 'react-redux'
+import 'leaflet/dist/leaflet.css'
+import L from 'leaflet'
```

Comment:
are we sticking with leaflet or will this be refactored to use OpenLayers at some point in the future?

Reply:
There's a ticket for switching to OpenLayer.